ksqldb-engine/src/main/java/io/confluent/ksql/util/SandboxedPersistentQueriesInSharedRuntimesImpl.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.util;
+
+/**
+ * Sandboxed {@link PersistentQueryMetadata} that prevents to modify the state of the internal
+ * {@link org.apache.kafka.streams.KafkaStreams}.
+ */
+public final class SandboxedPersistentQueriesInSharedRuntimesImpl
```

Comment:
Please see my other comment: checking the sandbox call path here, I still do not know why we need to have an extended class here: since start/stop are not triggered anyways in the sandboxing, can't we just create the `PersistentQueriesInSharedRuntimesImpl` even for sandboxing?